config/schema/strawberryfield.schema.yml

```diff
@@ -64,4 +64,18 @@ strawberryfield.storage_settings:
       label: 'Storage Scheme for Persisting Files'
     object_file_scheme:
       type: string
-      label: 'Storage Scheme for Persisting Digital Objects in JSON format'
\ No newline at end of file
+      label: 'Storage Scheme for Persisting Digital Objects in JSON format'
+      
+strawberryfield.archipelago_solr_settings:
+  type: config_object
+  label: 'Important Solr settings for Archipelago'
+  mapping:
+    type_server: 
```

Comment:
the naming of this properties is not totally accurate and we can make this better. Idea. For each setting, in this case the ADO type, we want them to have their own config map: So in this case the mapping would be, property name "ado_type", type "strawberryfield.archipelago_solr_settings.fieldconfig" and this `fieldconfig`type is a map that has the following properties:
- server: (read further down not needed really)
- index_id:
- field:

Regarding the "server" key. Indexes are entities. That means, that their names are unique, means also, the "server" here is more a commodity to allow people to fetch, via the UI, a given Server, but there is no need  to save it in the configuration, can be extracted from the same index. See https://github.com/esmero/format_strawberryfield/blob/8.x-1.0-beta1/src/Form/ViewModeMappingSettingsForm.php#L220, we never call the server directly, we use the index, fetch the server from it. Same can be done in the form here.

That way we can reuse that structure for other Solr fields we will want to allow people to configure.